### ISSUE TITLE
Displaying community curator contributions on WBPerson page

### ISSUE THREAD

**Issue by chris-grove** [WormBase/website#3957]
Something discussed at today's WB CIT call:
We'd like to display community curators' contributions on their respective WBPerson pages, perhaps in a new widget called "Curation Contributions" or something. We will be storing names of all community curators through our submission forms and will likely dump them out into ACEDB, although it isn't clear yet where or how.

@tharris  or  @sibyl229  Do either of you have any input / preference about how this is done at the ACEDB level? We were discussing having separate tables for each data type that people contribute (e.g. allele-phenotype, concise descriptions, etc.) considering the disparate nature of each type of data, but maybe it would be better consolidated.

**Comment by raymond91125** [comment]
one place that we display person evidence is concise description. 
perhaps we can expand on that mechanism.

raymond

**Comment by chris-grove** [comment]
@raymond91125   Right. I was thinking about this and wondering if we want a distinct mechanism from the "Person_evidence" route since the Community Curator identity is not necessarily a source of evidence, technically speaking, but rather a curator identity, like "Curator_confirmed" although I don't like that wording either.

**Comment by tharris** [comment]
Display:
I'd suggest NOT isolation these annotations I their own widget but instead, displaying them alongside everything else but perhaps with a special evidence hash flag. 

We can also display these on person pages as suggested. 

As for storage: do these need to be in acedb? Perhaps these should be modeled and stored in something new (@khowe has mentioned "gene-tomic" for example.

**Comment by chris-grove** [comment]
@tharris 
OK, I do think it would be good to have some sort of consolidated view of a user's contributions, for positive feedback/reinforcement, rather than distributed across many different places. I'm not sure what you mean by "alongside everything else": do you mean alongside the relevant data display?

Yeah, I think the Person pages makes sense for this, at least.

re: storage: I'm not particular about it being in ACEDB, but thought it may be more streamlined for everyone involved (?). I'm open to any other format that you might prefer. I'm not familiar with "gene-tomic"

**Comment by dasmoth** [comment]
"Genetomic" == working title for a system which contains (initially) the same data as geneace, using a new database platform.  It's probably not going to help with the display of user submissions in the near term.  In the future (if scope grows beyond geneace) then maybe yes.

Can I suggest the db mailing list (or maybe a ticket in the db github project) if there's interest in further discussion relating to new databases?

**Comment by tharris** [comment]
@chris-grove what I mean is that I believe that community submitted annotations should ultimately be treated exactly as regular curator-confirmed annotations.

For this example, there would simply be additional entires in the phenotype table of allele pages. There would be some sort of flag denoting the origin of the submission, just like we denote other forms of evidence.

And yep, totally, this should be displayed on Person pages. Since the form doesn't enforce logging in, this really means that somebody will need to confirm every submission. People tend to get upset if data or papers that aren't theirs end up displayed on their page :)

**Comment by chris-grove** [comment]
@tharris  OK, that all sounds good; and yes, there will be a confirmation process for these forms, so it shouldn't get all the way to the website without their consent
